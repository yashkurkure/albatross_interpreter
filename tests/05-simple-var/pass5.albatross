var variable string := "hello";

return 0;