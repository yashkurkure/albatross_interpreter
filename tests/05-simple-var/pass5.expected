Variable declared "variable" type string
